test/test_nn.py

```diff
@@ -2050,6 +2050,21 @@ def test_pad_scalar_error(self):
         self.assertRaises(AssertionError, lambda: F.pad(inputs, (1, 1)))
         self.assertRaises(AssertionError, lambda: F.pad(inputs, (1,)))
 
+    def test_load_error_msg(self):
+        expected_err_msg = (".*You can only torch.load from a file that is seekable. " +
```

Comment:
You should also try testing using a mock: https://github.com/pytorch/pytorch/blob/master/test/test_torch.py#L39-L69

Reply:
for some reason FilelikeMock cannot pass the is_real_file check https://github.com/weiyangfb/pytorch/blob/08101d0943223cd88980b260ece5cbfe057c957e/torch/serialization.py#L455